.clj-kondo/hooks/second_date/core.clj

```diff
@@ -0,0 +1,41 @@
+(ns hooks.second-date.core
+  (:require [clj-kondo.hooks-api :as hooks]))
+
+(defn- check-for-first-day-of-week-option [node]
```

Comment:
Since the new lib doesn't know about the `start-of-week` setting a few functions need you to explicitly pass it in, Kondo hook checks to make sure you do. Good to decouple this stuff anyway